src/main/java/appeng/core/AEConfig.java

```diff
@@ -70,7 +70,7 @@
 	public float spawnChargedChance = 0.92f;
 	public int quartzOresPerCluster = 4;
 	public int quartzOresClusterAmount = 15;
-	public int chargedChange = 4;
+	public final int chargedChange = 4;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Make this final field static too. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1170)